[2022-08-19 23:08:39,031] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: sefa_dag.get_telegram_version scheduled__2020-01-01T03:24:00+00:00 [queued]>
[2022-08-19 23:08:39,039] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: sefa_dag.get_telegram_version scheduled__2020-01-01T03:24:00+00:00 [queued]>
[2022-08-19 23:08:39,040] {taskinstance.py:1376} INFO - 
--------------------------------------------------------------------------------
[2022-08-19 23:08:39,043] {taskinstance.py:1377} INFO - Starting attempt 1 of 1
[2022-08-19 23:08:39,043] {taskinstance.py:1378} INFO - 
--------------------------------------------------------------------------------
[2022-08-19 23:08:39,071] {taskinstance.py:1397} INFO - Executing <Task(PythonOperator): get_telegram_version> on 2020-01-01 03:24:00+00:00
[2022-08-19 23:08:39,078] {standard_task_runner.py:52} INFO - Started process 2634 to run task
[2022-08-19 23:08:39,085] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'sefa_dag', 'get_telegram_version', 'scheduled__2020-01-01T03:24:00+00:00', '--job-id', '434', '--raw', '--subdir', 'DAGS_FOLDER/sefa_dag.py', '--cfg-path', '/tmp/tmp8cvku_kr', '--error-file', '/tmp/tmpvvqv00wg']
[2022-08-19 23:08:39,124] {standard_task_runner.py:80} INFO - Job 434: Subtask get_telegram_version
[2022-08-19 23:08:39,185] {task_command.py:371} INFO - Running <TaskInstance: sefa_dag.get_telegram_version scheduled__2020-01-01T03:24:00+00:00 [running]> on host fd7a784b9251
[2022-08-19 23:08:39,232] {taskinstance.py:1591} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=sefa_dag
AIRFLOW_CTX_TASK_ID=get_telegram_version
AIRFLOW_CTX_EXECUTION_DATE=2020-01-01T03:24:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2020-01-01T03:24:00+00:00
[2022-08-19 23:08:39,234] {python.py:173} INFO - Done. Returned value was: 1.24.0
[2022-08-19 23:08:39,254] {taskinstance.py:1420} INFO - Marking task as SUCCESS. dag_id=sefa_dag, task_id=get_telegram_version, execution_date=20200101T032400, start_date=20220819T230839, end_date=20220819T230839
[2022-08-19 23:08:39,296] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-08-19 23:08:39,318] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
